tests/integration/commands/test_ping.py

```diff
@@ -0,0 +1,25 @@
+import confluent_kafka
+from click.testing import CliRunner
+
+from esque import config
+from esque.cli.commands import ping
+from esque.topic_controller import TopicController
+
+
+def test_smoke_test_ping(cli_runner: CliRunner):
+    result = cli_runner.invoke(ping)
+
+    assert result.exit_code == 0
+
+
+def test_correct_amount_of_messages(
```

Comment:
This doesn't actually test the amount of messages

Reply:
Hm. Apparently I stopped halfway through. :D